daemon/images/mount.go

```diff
@@ -1,50 +0,0 @@
-package images
-
-import (
-	"context"
-	"fmt"
-	"runtime"
-
-	"github.com/containerd/log"
-	"github.com/docker/docker/container"
-	"github.com/pkg/errors"
-)
-
-// Mount sets container.BaseFS
-// (is it not set coming in? why is it unset?)
-func (i *ImageService) Mount(ctx context.Context, container *container.Container) error {
-	if container.RWLayer == nil {
-		return errors.New("RWLayer of container " + container.ID + " is unexpectedly nil")
-	}
-	dir, err := container.RWLayer.Mount(container.GetMountLabel())
-	if err != nil {
-		return err
-	}
-	log.G(ctx).WithFields(log.Fields{"container": container.ID, "root": dir, "storage-driver": container.Driver}).Debug("container mounted via layerStore")
-
-	if container.BaseFS != "" && container.BaseFS != dir {
-		// The mount path reported by the graph driver should always be trusted on Windows, since the
-		// volume path for a given mounted layer may change over time.  This should only be an error
-		// on non-Windows operating systems.
-		if runtime.GOOS != "windows" {
-			i.Unmount(ctx, container)
-			return fmt.Errorf("Error: driver %s is returning inconsistent paths for container %s ('%s' then '%s')",
```

Comment:
Oh! it was an existing error (still something to consider removing I guess)

Reply:
Removed